tree/30_generic_methods/_log_v3.cf

```diff
@@ -40,5 +40,5 @@ bundle agent _log_v3(message, class_parameter, old_class_prefix, class_prefix, a
       # Report using the appropriate bundle(s)
       "wrapper for ${configuration.enabled_loggers}"
         usebundle => ${configuration.enabled_loggers}("${message}", "${class_parameter}", "${old_class_prefix}", "${class_prefix}", "@{args}"),
-        comment   => "Call the ${method} bundle with arguments ${old_class_prefix}, ${class_prefix}, @{args}";
+        comment   => "Call the ${method} bundle with arguments ${old_class_prefix}, ${class_prefix}, @{args}, ${report_data.directive_id}";
```

Comment:
Can you chang that to
Call the ${method} bundle for directive ${report_data.directive_id}, with arguments ${old_class_prefix}, ${class_prefix}, @{args}